.env.demo

```
@@ -223,3 +223,5 @@ RESEND_API_KEY=re_xxxxxxxxxx
 
 # Prisma log type. Default set to error
 PRISMA_LOGS = error
+# DB_ALERT_ENABLE=
+# DB_ALERT_EMAILS=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)